src/patternfly/demos/Toolbar/examples/toolbar-complex-example.hbs

```diff
@@ -0,0 +1,99 @@
+{{#> toolbar pf-l-toolbar--modifier="pf-u-justify-content-space-between pf-u-mx-xl pf-u-my-md"}}
+    {{#> toolbar-section aria-label="First section"}}
+        {{#> toolbar-group}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-2xl"}}
+                {{#> input-group}}
+                    {{#> dropdown id="dropdown-example-collapsed"}}
+                    All
+                    <i class="fas fa-caret-down pf-c-dropdown__toggle-icon"></i> 
+                    {{/dropdown}}
+                    {{#> input-group-action input-group-action--modifier="pf-m-align-right" inputGroupTextType="button" aria-label="popover for input"}}
```

Comment:
I think this should be a label instead of a button.

Reply:
fixed